overlord/configstate/applyconfighandler.go

```diff
@@ -21,7 +21,7 @@ package configstate
 
 import "github.com/snapcore/snapd/overlord/hookstate"
 
-// applyConfigHandler is the handler for the apply-config hook.
+// applyConfigHandler is the handler for the configure hook.
```

Comment:
Tracked this one in the "Configuration support" card:

https://trello.com/c/PpM4XRUQ/30-configuration-support